daft/sql/sql_scan.py

```diff
@@ -90,11 +82,11 @@ def to_scan_tasks(self, pushdowns: Pushdowns) -> Iterator[ScanTask]:
             )
             sql = f"SELECT * FROM ({self.sql}) AS subquery WHERE {left_clause} AND {right_clause}"
             stats = Table.from_pydict({self._partition_col: [partition_bounds[i], partition_bounds[i + 1]]})
-            file_format_config = FileFormatConfig.from_database_config(DatabaseSourceConfig(sql=sql))
+            file_format_config = FileFormatConfig.from_database_config(DatabaseSourceConfig(sql, self.conn))
 
             scan_tasks.append(
                 ScanTask.sql_scan_task(
-                    url=self.url,
+                    path=repr(self.conn),
```

Comment:
We should provide an explicit `.url` property instead of relying on `repr`, which is more for human-readability and display